docs/userGuide/contentAuthoring.md

```diff
@@ -125,6 +125,29 @@ You can insert variables to be replaced by pre-defined values anywhere in your s
 
 Some helpful variables are provided by MarkBind for your convenience, as shown below. 
 
+#### Font Awesome Icons
+
+This asset is provided by [Font Awesome](https://fontawesome.com/) under their [free license](https://fontawesome.com/license).
+<br><br>
+<tip-box>
+Traditional usage:  
+`<span class='fas fa-chess-knight'></span>`
+<br>  
+MarkBind usage:  
+<code>{<span></span>{fas_chess_knight}}</code>
+</tip-box>
+
+Font Awesome has three different styles for their free icons, each with their own prefix:
+<strong>Solid</strong> (`fas`), <strong>Regular</strong> (`far`) and <strong>Brands</strong> (`fab`).<br>
+Be sure to check Font Awesome's [full list of icons](https://fontawesome.com/icons?d=gallery&m=free) on which prefix to use.<br>
+
+Note that you will have to shorten the icon's name before using it within MarkBind. It is a two step process: 
+1. Shift the icon's prefix to replace `fa`, the icon's name first two characters. e.g `fas fa-chess-knight` becomes `fas-chess-knight`
+2. Next, replace all instances of `-` with `_` in the icon's name. e.g. `fas-chess-knight` becomes `fas_chess_knight`
+
+After doing so, you are able to use the icons anywhere on your site by surrounding an icon's shortened name with double curly braces.<br>
+e.g. <code>{<span></span>{fas_fighter_jet}}</code> will be rendered as {{fas_fighter_jet}}
+
 #### Glyphicons
 
 This asset is provided by [Glyphicons](https://glyphicons.com/) via Bootstrap.

```

Comment:
Let's specify the version number of Bootstrap here since there's now multiple versions of Bootstrap floating around.

"Bootstrap" -> "Bootstrap 3".